mongodbatlas/data_source_mongodbatlas_project.go

```diff
@@ -91,6 +139,11 @@ func dataSourceMongoDBAtlasProjectRead(ctx context.Context, d *schema.ResourceDa
 		return diag.FromErr(fmt.Errorf("error getting project's teams assigned (%s): %s", projectID, err))
 	}
 
+	apiKeys, err := getProjectApiKeys(conn, ctx, project.OrgID, project.ID)
+	if err != nil {
+		return diag.FromErr(fmt.Errorf("error getting project's api keys (%s): %s", projectID, err))
```

Comment:
@pitakill you can use `diag.Errorf("error getting project's api keys (%s): %s", projectID, err)` instead of using `diag.FromErr(fmt.Errorf....`

for more context, you can check https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html#support-for-diagnostics

Reply:
done https://github.com/mongodb/terraform-provider-mongodbatlas/pull/504/commits/50f1664f65d5aa8ccf672e3dabbb66d8118adeea
also adjusted it in other places in this file to keep it consistent